modules/gdnative/pluginscript/pluginscript_script.cpp

```diff
@@ -252,6 +252,13 @@ Error PluginScript::reload(bool p_keep_state) {
 			(godot_string *)&_source,
 			(godot_error *)&err);
 	if (err) {
+		// Manifest's attributes must be explicitly freed
```

Comment:
There's another early return at line 279 (`ERR_FAIL_V_MSG`) which would leak similarly I guess?

Reply:
good catch ! I've updated the PR ;-)